docs/docs/reference/sdk/search/useSearch.mdx

```diff
@@ -0,0 +1,115 @@
+import PropsComponent from "@site/src/components/PropsComponent/PropsComponent";
+
+# useSearch
+
+This hook allows you to consume `SearchContext`. This context allows you to apply changes to the search state as well as consume its values
+
+## Import
+
+```tsx
+import { useSearch } from '@faststore/sdk'
+```
+
+## Usage
+
+```tsx
+// Create a selection sort button
+
+const keyMap = {
+  price_desc: 'Price, descending',
+  price_asc: 'Price, ascending',
+  orders_desc: 'Top sales',
+  name_asc: 'Name, A-Z',
+  name_desc: 'Name, Z-A',
+  release_desc: 'Release date',
+  discount_desc: 'Discount',
+  score_desc: 'Relevance',
+}
+
+const keys = Object.keys(keyMap)
+
+function Component () {
+  const {
+    setSort,
+    state: { sort },
+  } = useSearch()
+
+  return (
+    <select
+      onChange={(e) => setSort(keys[e.target.selectedIndex])}
+      value={sort}
+    >
+      {keys.map((key) => (
+        <option key={key} value={key}>
+          {keyMap[key]}
+        </option>
+      ))}
+    </select>
+  )
+}
+```
+
+## Return
+
+<PropsComponent
+    name="itemsPerPage"
+    type="number"
+    description="Number of items in a search page"
+/>
+
+<PropsComponent
+    name="state"
+    type="SearchState"
+    description="Current state of search"
```

Comment:
```suggestion
    description="Current state of the facets."
```

Reply:
Not only the facets, but also sort, basePath etc

Reply:
Is it correct now?